WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRowViewModel.swift

```diff
@@ -71,6 +76,7 @@ struct InboxNoteRowViewModel: Identifiable, Equatable {
         self.siteID = siteID
         self.stores = stores
         self.isPlaceholder = isPlaceholder
+        self.isRead = isRead
     }
 
     static func == (lhs: InboxNoteRowViewModel, rhs: InboxNoteRowViewModel) -> Bool {
```

Comment:
I wonder if the issue about reloading the style for the row can be fixed by updating this method. We'll need to check for the `isRead` status of the models to make sure that it doesn't change; otherwise the view will need to reload to reflect the change.

Reply:
I'll try to check that in the next PR for fixing the issue about the reload 🙇

Reply:
And... this was the issue, fixed here cd3d1d75a73857f1fc9c71cf923e7efd95fcc5b7 🙇 good catch @itsmeichigo!